### PR TITLE
sys-fs/udiskie: Proxy maintainership, version bump, python 3 support.

### PR DESCRIPTION
**Pull request by sdlarsen**

### CODE REVIEW

File: sys-fs/udiskie/udiskie-1.1.1.ebuild [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=6
```
Comment (Zlogene) [inline sys-fs/udiskie/udiskie-1.1.1.ebuild]: no, you should not change EAPI of stable ebuilds.

File: profiles/repo_name [hunk 2]
```diff
@@ -1 +1 @@
-gentoo
+local-gentoo
```
Comment (hydrapolic) [inline profiles/repo_name]: Are you sure?

Comment (sdlarsen) [reply profiles/repo_name]: No, my bad. Removed.

File: sys-fs/udiskie/udiskie-1.7.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE=""
+
+RDEPEND="dev-python/docopt[${PYTHON_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	dev-python/pyxdg[${PYTHON_USEDEP}]
+	dev-python/pyyaml[${PYTHON_USEDEP}]
+	sys-fs/udisks:2"
+DEPEND="app-text/asciidoc
+	dev-python/setuptools[${PYTHON_USEDEP}]"
```
Comment (mgorny) [inline sys-fs/udiskie/udiskie-1.7.5.ebuild]: It uses `entry_points` in `setup.py`, so setuptools needs to be an RDEPEND as well.

File: sys-fs/udiskie/udiskie-1.7.5.ebuild [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+DESCRIPTION="An automatic disk mounting service using udisks"
+HOMEPAGE="https://pypi.org/project/udiskie/ https://github.com/coldfix/udiskie"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE=""
+
+RDEPEND="dev-python/docopt[${PYTHON_USEDEP}]
```
Comment (mgorny) [inline sys-fs/udiskie/udiskie-1.7.5.ebuild]: `setup.py` also lists `gbulb` as a requirement. Are you sure it works without it?

File: sys-fs/udiskie/udiskie-1.7.5.ebuild [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	dev-python/pyxdg[${PYTHON_USEDEP}]
+	dev-python/pyyaml[${PYTHON_USEDEP}]
+	sys-fs/udisks:2"
+DEPEND="app-text/asciidoc
```
Comment (mgorny) [inline sys-fs/udiskie/udiskie-1.7.5.ebuild]: Missing DEPEND on package providing `/usr/bin/msgfmt` which is used in `setup.py`.

File: sys-fs/udiskie/udiskie-1.7.5.ebuild (hunk 3 above)
Comment (sdlarsen) [reply sys-fs/udiskie/udiskie-1.7.5.ebuild]: True, will fix.

File: sys-fs/udiskie/udiskie-1.7.5.ebuild (hunk 4 above)
Comment (sdlarsen) [reply sys-fs/udiskie/udiskie-1.7.5.ebuild]: Actually it does for me, but I wonder why - obviously I don't get in contact with that code with my use of udiskie. I see gbulb requires python 3.5+, so 3.4 needs to go. I guess I'll have to package gbulb as well then :)

File: sys-fs/udiskie/udiskie-1.7.5.ebuild (hunk 5 above)
Comment (sdlarsen) [reply sys-fs/udiskie/udiskie-1.7.5.ebuild]: That looks like gettext - added.

File: sys-fs/udiskie/udiskie-1.7.5.ebuild (hunk 4 above)
Comment (mgorny) [reply sys-fs/udiskie/udiskie-1.7.5.ebuild]: FWICS it's used in `cli.py`.

**Comment by mgorny** [comment]
Oh, I see that upstream has some tests. Please add `python_test` to run https://wiki.gentoo.org/wiki/Project:Python/Tests for tips.

**Comment by mgorny** [comment]
Ping.

**Pull request closed without merging**